app/core/serializers.py

```diff
@@ -55,10 +55,16 @@ def create(self, validated_data):
                     'user_exists_error': True
                 }
                 raise res
-            user = get_user_model().objects.create_user(email=user_info.get('email'), password=user_info.get('password'))
-            UserSerializer(data=user_info).update(user, user_info)
-        recruiter = Recruiter.objects.create(user=user, company=validated_data['company'], is_admin=validated_data['is_admin'])
-        recruiter.save()
+            user = get_user_model().objects.create_user(**user_info)
+            recruiter = Recruiter.objects.create(user=user, **validated_data)
+        else:
+            res = serializers.ValidationError()
+            res.status_code = HTTP_400_BAD_REQUEST
+            res.detail = {
```

Comment:
Let's move such constant `detail` dictionaries to a file `error_messages.py` and then raise error like:
```
raise serializers.ValidationError(detail=MISSING_USER_MESSAGE, status_code=HTTP_400_BAD_REQUEST)
```

Reply:
Updated 👍